crates/turborepo/src/main.rs

```diff
@@ -1,20 +1,47 @@
-use std::{env::current_exe, process, process::Stdio};
+use std::{
+    env::{consts, current_exe},
+    process,
+    process::Stdio,
+};
 
 use anyhow::Result;
 use dunce::canonicalize as fs_canonicalize;
+use log::{debug, error};
 use turborepo_lib::{Args, Payload};
 
 fn run_go_binary(args: Args) -> Result<i32> {
-    // canonicalize the binary path  to ensure we can find go-turbo
-    let mut go_binary_path = fs_canonicalize(current_exe()?)?;
+    // canonicalize the binary path to ensure we can find go-turbo
+    let turbo_path = fs_canonicalize(current_exe()?)?;
+    let mut go_binary_path = turbo_path.clone();
     go_binary_path.pop();
     #[cfg(windows)]
     go_binary_path.push("go-turbo.exe");
     #[cfg(not(windows))]
     go_binary_path.push("go-turbo");
 
-    let serialized_args = serde_json::to_string(&args)?;
+    if !go_binary_path.exists() {
```

Comment:
I'm confused, if the go binary doesn't exist we log out that we found it?

Reply:
Big oof. Was using this to test the error message format without removing my local go-bin. Thanks.